packages/js/github-actions/actions/prepare-extension-release/src/woo-extension-create-pr-for-release.mjs

```diff
@@ -19,9 +19,15 @@ export default async ( {
 		( wcVersion ? ' --wc_tested=' + wcVersion : '' );
 
 	const title = `${ type } ${ version }`;
+	// We need to add only one newline before the pre-steps to make sure it's rendered as a list.
+	let trimmedPreSteps = preSteps;
+	if ( trimmedPreSteps !== '' ) {
+		trimmedPreSteps = '\n' + trimmedPreSteps.trim();
+	}
+
 	const body = `## Checklist
 1. [ ] Check if the version, base, and target branches are as you desire.
-1. [ ] Make sure you have \`woorelease\` installed and set up.
+1. [ ] Make sure you have \`woorelease\` installed and set up.${ trimmedPreSteps }
 1. [ ] Simulate the release locally
    \`\`\`sh
    git fetch origin ${ refName }
```

Comment:
Just curious why these `git fetch` and `git checkout` lines are added here. Those are needed if we want to update our local copy, but not needed for woorelease since it does a separate clone/checkout to the tmp folder. Not bad to leave but just wonder if it needs additional clarification as those commands need to be run specifically in a local repo copy.

Reply:
Good point. I kept them here, as some repos require additional steps to perform on the repo like GLA requires manual removal of entries from `changelog.txt`. 
I wonder what's better:
- remove it so other repos will have a cleaner process
- keep it, it's copied together with the simulate anyway and does no harm
- automate the removal of changelog.txt :godmode:

Reply:
> remove it so other repos will have a cleaner process

If we remove it from here, for additional steps, we will not have the possibility to provide the exact `refName` (unless we introduce a templating engine, which IMO is overkill, given our goal is to simplify and unify steps to be same for all repos)

Reply:
Addressed somewhat in https://github.com/woocommerce/grow/pull/58/commits/6256683d661e05461ef7fa2a8023b8915591181d